ksqldb-common/src/test/java/io/confluent/ksql/schema/ksql/LogicalSchemaTest.java

```diff
@@ -44,15 +44,23 @@
 import io.confluent.ksql.schema.ksql.types.SqlDecimal;
 import io.confluent.ksql.schema.ksql.types.SqlTypes;
 import io.confluent.ksql.schema.utils.FormatOptions;
+import io.confluent.ksql.util.KsqlConfig;
 import io.confluent.ksql.util.KsqlException;
 import java.util.List;
 import java.util.Optional;
 import org.apache.kafka.connect.data.Schema;
+import org.junit.Rule;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
 
 @SuppressWarnings({"UnstableApiUsage", "unchecked"})
 public class LogicalSchemaTest {
 
+  @Rule
+  public final MockitoRule rule = MockitoJUnit.rule();
```

Comment:
nit: use `@RunWith(MockitoJUnitRunner.class)` instead to be consistent with all the other test files using Mockito in this codebase.